azure-pipelines.yml

```diff
@@ -181,7 +181,7 @@ stages:
               eval ${CMD}
             displayName: 'Build python $(python.version) image'
             env:
-              FULL_BRANCH_NAME: $(Build.SourceBranch)
+              FULL_BRANCH_NAME: $(Build.SourceBranchName)
```

Comment:
We often name our branches something like `m/GREAT-1234/branch-desc`. In this case, `SourceBranchName` is set to `branch-desc` and, since it is not really the actual branch name, which will prevent us from pulling the correct branch and the build will fail.